Loop/Managers/LoopAppManager.swift

```diff
@@ -208,6 +210,12 @@ class LoopAppManager: NSObject {
     }
 
     // MARK: - Remote Notification
+    
```

Comment:
Pete - Since you looked over this before I realized I should have this feature flag check. At least that is what is done in dev now.